drivers/SmartThings/matter-switch/src/test/test_matter_switch.lua

```diff
@@ -548,4 +548,70 @@ test.register_message_test(
   }
 )
 
+test.register_message_test(
+  "Min and max color temperature attributes set capability constraint",
+  {
+    {
+      channel = "matter",
+      direction = "receive",
+      message = {
+        mock_device.id,
+        clusters.ColorControl.attributes.ColorTempPhysicalMinMireds:build_test_report_data(mock_device, 1, 500)
```

Comment:
The min value should be less than the max value. I suggest using 153 and 555 which should translate to 6500 K and 1800 K respectively.

Reply:
I updated the test to use these values in 7be32c0a41199c9d51ee83463fc6dd9800c38304 as suggested.